AndroidSDK/src/com/leanplum/internal/Request.java

```diff
@@ -69,6 +68,8 @@
   private static final Map<String, Boolean> fileTransferStatus = new HashMap<>();
   private static int pendingDownloads;
   private static NoPendingDownloadsCallback noPendingDownloadsBlock;
+  private static boolean moreThenOneRequestToServer = false;
```

Comment:
maybe we don't store this on the class level, since it seems only be used once? So I suggest initializing it where its used to avoid to much jumping within the code :)